src/main/views/manage-users.njk

```diff
@@ -0,0 +1,56 @@
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% extends "template.njk" %}
+
+{% block beforeContent %}
+  {{ govukBreadcrumbs({
+    collapseOnMobile: true,
+    items: [
+      {
+        text: "Home",
+        href: "/"
+      },
+      {
+        text: "Manage users"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">Manage Users</h1>
+      <form method="GET" action="/user-results">
+        {{ govukInput({
+          label: {
+            text: "Search for an existing user",
```

Comment:
I think it'd be potentially good to switch to using strings loaded from a locale file

Reply:
The idam user dashboard will only need English text since it is an internal application. We can discuss if we need to implement the locale file and create a separate task for that?

Reply:
Yep! Let’s discuss and potentially implement in a separate PR instead :)